system-functionalities.md

```diff
@@ -307,6 +369,9 @@ For now, simple clear-text storage is acceptable.
 
 #### Server-side storage
 - [TODO #28](https://github.com/boltlabs-inc/key-mgmt-spec/issues/28). Include appropriate requirements for server-side secure storage and generate relevant issues in key-mgmt.
+- All data stored by the server MUST be encrypted by a key known only to the server.
```

Comment:
Re: "a key known only to the server" part...are we going to provide guidance on the type of key (ie symmetric vs asymmetric) as part of this PR or another?

Reply:
This is part of TODO #28, not this PR. I just added a line that was relevant to the current PR's topic.